src/components/ui/debounced-input.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+export function DebouncedInput({
+  value: initialValue,
+  onChange,
+  debounce = 500,
+  ...props
+}: {
+  value: string | number;
```

Comment:
to keep type definitions more organized, could we define prop types inside an interface?